view/frontend/web/js/model/spi/callbacks/on-create-payment-order-callback.js

```diff
@@ -34,6 +34,9 @@ define(
 
             if (!isSpiContainer && addressProvided) {
                 if (isWalletPayment) {
+                    if (!paymentData['billing_address']['phoneNumber']) {
+                        paymentData['billing_address']['phoneNumber'] = paymentData['shipping_address']['phoneNumber'];
```

Comment:
Are we sure about this? The shipping address and billing address can be different people in the case of a gift for example.